device-posture/device-posture.https.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+//<script src="../resources/orientation-event-helpers.js"></script>
+<script>
+'use strict';
+
+async function cleanup() {
+  await test_driver.set_device_posture('continuous');
+}
+
+promise_test(async (t) => {
+  t.add_cleanup(cleanup);
+  const promise = new Promise(resolve => {
```

Comment:
Use an EventWatcher instead.

Reply:
This is not the code but in this case what I should use in EventWatcher?

const watcher = new EventWatcher(t, navigator.devicePosture, ['change']);
await test_driver.set_device_posture('folded');
watcher.wait_for('change').then(() => {
  console.log(navigator.devicePosture.type);
  watcher.stop();
});

Witrh above code I get "assert_true: Not expecting event, but got change event expected true got false".

Reply:
I think it might be more like this

  const watcher = new EventWatcher(t, navigator.devicePosture, ['change']);
  /* **await removed** */ test_driver.set_device_posture('folded');
  **await** watcher.wait_for('change').then(() => {
    console.log('JV666 1700 ' + navigator.devicePosture.type);
  });

Reply:
EventWatcher used in https://github.com/JuhaVainio/WPT/pull/4/commits/b1b8d4766e1c998e8e4e41a78a97ce5bf6f4e2c3